nx-ignore@16.1.1  Error: Cannot find module '@swc/helpers/_/_extends'

When running npx nx-ignore MY_APP with nx-ignore@16.1.1  an error is thrown.

Vercel:
13:44:05.321 | Running "npx nx-ignore pdf-generator"
13:44:06.240 | npm WARN exec The following package was not found and will be installed: nx-ignore@16.1.1
13:44:06.643 | node:internal/modules/cjs/loader:1029
13:44:06.643 | throw err;
13:44:06.643 | ^
13:44:06.643 |  
13:44:06.643 | Error: Cannot find module '@swc/helpers/_/_extends'
13:44:06.643 | Require stack:
13:44:06.643 | - /vercel/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js
13:44:06.643 | at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
13:44:06.643 | at Function.Module._load (node:internal/modules/cjs/loader:871:27)
13:44:06.643 | at Module.require (node:internal/modules/cjs/loader:1098:19)
13:44:06.643 | at require (node:internal/modules/cjs/helpers:108:18)
13:44:06.644 | at Object.<anonymous> (/vercel/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js:3:18)
13:44:06.644 | at Module._compile (node:internal/modules/cjs/loader:1196:14)
13:44:06.644 | at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
13:44:06.644 | at Module.load (node:internal/modules/cjs/loader:1074:32)
13:44:06.644 | at Function.Module._load (node:internal/modules/cjs/loader:909:12)
13:44:06.644 | at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
13:44:06.644 | code: 'MODULE_NOT_FOUND',
13:44:06.644 | requireStack: [
13:44:06.644 | '/vercel/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js'
13:44:06.644 | ]
13:44:06.644 | }

Locally:
andersbs@mac xpand-frontend % npx nx-ignore pdf-generator
node:internal/modules/cjs/loader:1024
  throw err;
  ^

Error: Cannot find module '@swc/helpers/_/_extends'
Require stack:
- /Users/andersbs/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1021:15)
    at Function.Module._load (node:internal/modules/cjs/loader:866:27)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/Users/andersbs/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js:3:18)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/andersbs/.npm/_npx/cf7fc04e3df2a243/node_modules/nx-ignore/src/index.js'
  ]
}


This has been working perfectly with version 15.

--- a/packages/nx-ignore/package.json
+++ b/packages/nx-ignore/package.json
@@ -20,6 +20,6 @@
   },
   "dependencies": {
     "fs-extra": "^10.1.0",
-    "@swc/helpers": "^0.4.14"
+    "@swc/helpers": "0.5.1"
   }
 }

